frontend/javascripts/router.js

```diff
@@ -427,10 +427,7 @@ class ReactRouter extends React.Component<Props> {
               />
               <Route
                 path="/auth/register"
```

Comment:
Would you mind renaming the route to `/auth/signup`?

Reply:
Should we keep the old `register` path (and redirect to `signup`)? Or can it be a "breaking change"?

Reply:
should be a redirect or keep both.

Reply:
done